.github/workflows/connector-ci-checks.yml

```diff
@@ -186,6 +186,30 @@ jobs:
     needs: [jvm-connectors-test-matrix]
     runs-on: ubuntu-24.04
     steps:
+      - name: Authenticate as GitHub App
+        uses: actions/create-github-app-token@v1
+        id: get-app-token
+        if: github.event_name == 'workflow_call' && inputs.gitref != ''
```

Comment:
This step, and the below will not run when this is triggered via a normal pull request trigger. It will only run from slash commands, i.e. in cases where `github.event_name == 'workflow_call' && inputs.gitref != '' `